src/hashbang.html

```diff
@@ -61,7 +69,7 @@
     <script src='/assets/icon.js'></script>
     <h1>#!</h1>
     <a href="view-source:https://hashbang.sh">
-      <code>curl hashbang.sh | gpg > hashbang.sh</code>
+      <code>curl hashbang.sh | gpg | sh</code>
```

Comment:
Half the point of the message is to indicate that you -shouldn't- run it using a oneliner; `curl hashbang.sh | gpg > hashbang.sh; vim hashbang.sh` then when you're satisfied with the file, `sh hashbang.sh`.